public/style.css

```diff
@@ -226,7 +226,6 @@ background: url(fitSlider.gif) no-repeat right 3px;
 background-size: auto;
 box-sizing: border-box;
 display: block;
-font-family: Calibre, "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
 font-size: 13px;
 font-style: normal;
 font-weight: 400;
```

Comment:
After this line, there are a few styles you are creating which aren't necessary. The ones that say 0px, normal, or any of these default values, should be removed so your stylesheet is not as overwhelming.